### ISSUE TITLE
Cannot rename original Renovate MR to trigger replacement onboarding MR

### ISSUE THREAD

**Issue by mcritchlow** [renovatebot/config-help#730]
### Which Renovate are you using?

WhiteSource Renovate App

### Which platform are you using?

GitLab.com

### What would you like to do?

We haven't had MRs for a couple repositories in a while, and I suspect something is wrong with our configuration. [After reading the docs](https://docs.renovatebot.com/reconfigure-renovate/), I decided the easiest path forward was to delete our `renovate.json` and start from scratch with a new onboarding MR that I could easily test configuration changes against.

Unfortunately, it seems that the second step in that process (renaming the original MR) is not possible. I get a 500 error in Gitlab when trying to do so.

Here is the MR I'm trying to rename: https://gitlab.com/ucsdlibrary/development/hifive/-/merge_requests/23

Thanks for any advice, and the great product!

**Comment by rarkins** [comment]
Maybe try adding back your renovate.json and we work out why there's no MRs?

**Comment by viceice** [comment]
Please provide to be debug logs. 

You also can show us your config, so we can check it for issues.

For the mr rename issue you need to ask the gitlab owner for help, as we can't change that.

**Comment by mcritchlow** [comment]
Hi, thanks for the quick feedback!

> Maybe try adding back your renovate.json and we work out why there's no MRs?

Done. I've setup an MR and one of my teammates will review soon and get that merged. 

> Please provide to be debug logs.

Unfortunately, someone higher up than me setup our Renovate account and he's on vacation this week. I'll ask him to login and provide any logs as soon as he's back.

> For the mr rename issue you need to ask the gitlab owner for help, as we can't change that.

I'm an Owner on our gitlab org, and I don't see any feature/toggle about renaming closed MRs. But I'll keep looking!

Thank you both, again, for the quick feedback.

**Comment by mcritchlow** [comment]
@rarkins @viceice - following up that, unfortunately adding back the renovate.json file didn't seem to trigger any new MRs from RenovateBot.

And, if I go into the dashboard, I don't see any log information for the noted repository:
![image](https://user-images.githubusercontent.com/67506/88227339-511c8780-cc22-11ea-8127-9d50ef5e3a6f.png)

Here is the re-committed configuration file: https://gitlab.com/ucsdlibrary/development/hifive/-/blob/trunk/renovate.json

The only thing I can think of, is that we recently changed the base branch for most of our repos from `master` to `trunk`. I figured this might be an issue, but another project which make the same change seems to still receive MR and has log entries as current as an hour ago: https://gitlab.com/ucsdlibrary/development/dmr

Thoughts?
